packages/cache/tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "baseUrl": "./",
     "outDir": "./lib",
     "rootDir": "./src",
-    "lib": ["es6", "dom"]
+    "lib": ["es6", "dom"],
+    "useUnknownInCatchVariables": false
```

Comment:
Added this because the ts compiler raised issues with the catch block err variable having no type defined. Use of  `Error`, `any` and `unknown` was also not allowed.